benchmark/lib/report.dart

```diff
@@ -36,14 +38,12 @@ pb.Platform createPlatform() {
 get _isDartVM => !identical(1, 1.0);
 
 final bool _checkedMode = () {
-  bool x = true;
-  try {
-    // Trigger an exception if we're in checked mode.
-    x = "" as dynamic;
-    return x != ""; // return false; suppress unused variable warning
-  } catch (e) {
+  var checked = false;
+  assert(() {
```

Comment:
This is not checking the same thing.

The code as originally written:
 - in Dart 1 was checking for --checked.
 - in Dart 2 was checking for --omit-implicit-checks when compiled with dart2js

Now it is checking for --enable-asserts.

So I expect a performance regression on 'production' apps.

Reply:
@rakudrama – suggestions?

Reply:
revert this part

Reply:
And document it better.